src/app/mod.rs

```diff
@@ -475,6 +497,8 @@ impl App {
             BookmarkDel => self.delete_bookmark(),
             BookmarkEditTags => self.edit_bookmark_tags(),
             ClearCache => self.clear_cache(),
+            ClickNextPage => self.server_send(InnerMessage::ClickNextPage()),
```

Comment:
I think in general the app structure is due for a cleanup, but I don't have a strong opinion at the moment on how. I'll work on this a bit later when I'm more free.